zombies.js

```diff
@@ -79,6 +90,22 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed){
+  const pack = [];
+  this.health = health;
+  const maxHealth = this.health;
```

Comment:
Please try to keep like variables together. So the pack and maxHealth should be next to each other at the top of the function.